composer.json

```diff
@@ -17,8 +17,9 @@
         "php": ">=8.0",
         "ext-json": "*",
         "ext-redis": "*",
-        "utopia-php/framework": "0.*.*",
-        "utopia-php/cli": "0.15.*"
+        "utopia-php/framework": "0.33.*",
+        "utopia-php/cli": "0.15.*",
+        "utopia-php/queue": "0.7.*"
```

Comment:
Next step would also be to add realtime here. We can do it when we start using utopia/di in all libraries.

Reply:
Sure, but that will take time

Reply:
NP, I have a POC being tested.